src/client.rs

```diff
@@ -1976,13 +1976,27 @@ impl LoginConfigHandler {
     ///
     /// # Arguments
     ///
-    /// * `value` - The view style to be saved.
+    /// * `value` - The scroll style to be saved.
     pub fn save_scroll_style(&mut self, value: String) {
         let mut config = self.load_config();
         config.scroll_style = value;
         self.save_config(config);
     }
 
+    /// Save edge scroll edge thickness to the current config.
+    ///
+    /// # Arguments
+    ///
+    /// * `value` - The edge thickness to be saved.
+    pub fn save_edge_scroll_edge_thickness(&mut self, value: i32) {
+        self.set_option("edge_scroll_edge_thickness".to_string(), value.to_string());
```

Comment:
t is recommended to add a field in the `PeerConfig` of  crate `hbb_common` instead of using `options`. You can refer to `trackpad_speed`, 

https://github.com/rustdesk/rustdesk/blob/b75f4daa47bd8899549d15a22defe65bdfc28d6f/src/client.rs#L2408

https://github.com/rustdesk/hbb_common/blob/a4053b929b14059b1bd116900de8a103d9d838ae/src/config.rs#L331.

Reply:
Okay, that's cool. I was leaning toward doing that but previous advice in a related area had been to not add dedicated fields but instead use generic name/value functionality. But, I definitely prefer having a field for it because then it sits alongside `scroll_style` in the serialized form, much neater. :-) Will switch.

Reply:
Alright I have it all sorted locally. The `rustdesk` code, of course, can't work without the addition of the field to `hbb_common`, so I haven't yet pushed up the commit here. I'm waiting for the `hbb_common` change to go in before doing that. Let me know if I shouldn't wait and I'll go ahead and push this side of the changes too. 🙂

Reply:
Thinking it through, the only reason to not push the changes is that you can't build/test this branch without the `hbb_common` changes, but you can just go into the submodule repository and fetch the commit and check it out, so there's really no reason not to.